From 6b78e3280032660418f02b890897e7018f7252bb Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Thu, 11 Jul 2024 20:35:49 +0200
Subject: [PATCH] arm64: dts: st: add wakeup-source for wake-up button on
 stm32mp257-ev1 board

Add the missing "wakeup-source" field for wake-up button . This indication
is required to exit of the S2Idle loop for freeze command.

Change-Id: If5b92319261d1d101b2064d74675e9c977b710dd
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/393829
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -83,6 +83,7 @@
 			label = "wake-up";
 			linux,code = <KEY_WAKEUP>;
 			interrupts-extended = <&optee 0>;
+			wakeup-source;
 			status = "okay";
 		};
 	};
